doc/articles/features/windows-ui-startscreen.md

```diff
@@ -0,0 +1,13 @@
+# Uno Support for Windows.UI.StartScreen
```

Comment:
Could you add this to toc.yml https://github.com/unoplatform/uno/blob/master/doc/articles/toc.yml#L65 - that way it shows up in the [generated docs](https://platform.uno/docs/articles/intro.html) sidebar.